### PR TITLE
Nanosuit armor slowdown removed, quick cuff breaking, bug fixes and code cleanup.

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
add: Nanosuit can now quickly break cuffs in strength mode 
balance: Armor mode in nanosuit no longer slows you down
balance: Increased nanosuit damage to simple mobs and monkeys, aliens also take a bit of stamina damage
fix: You can no longer use the nanosuit's head stomp while laying down
fix: Attacking simple mob's in nanosuit's cloak now properly disables it
/:cl:

[why]: I feel like the slowdown penalty for the nanosuit punishes the user for no reason while strength mode felt inconsistent being able to beat up everyone and increase damage with melee objects but not let you burst out of cuffs like the hulk. Also the melee damage on simple mobs was weak. Code improvements were done to make the file more sane to read for reviewers. There were also some pretty dumb things going on like using `usr` instead of `user`, hard-coded radial menus (ew) and low end procs such as `Move()` and `Stat()` having pretty gross overrides on them.

### CODE REVIEW

**Comment by k-k-karma** [comment]
HOLY SHIT YES

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -647,7 +647,7 @@
 			to_chat(C, "<span class='unconscious'>You feel a breath of fresh air... which is a sensation you don't recognise...</span>")
 
 /mob/living/carbon/human/cuff_resist(obj/item/I)
-	if(dna && dna.check_mutation(HULK))
+	if(dna && dna.check_mutation(HULK) || istype(mind.martial_art, /datum/martial_art/nanosuit)) //Hippie edit -- Allows nanosuit strength mode to break cuffs fast
```
Comment (k-k-karma) [inline code/modules/mob/living/carbon/human/human.dm]: believe this should be modularizable if you put the proc in it's own section and call the parent

Comment (yoyobatty) [reply code/modules/mob/living/carbon/human/human.dm:650]: I've tried it endlessly months ago without success, this is my only solution

**Comment by earthmoonjune** [comment]
+1

**Comment by Manly-Dwarf** [comment]
okay +1

**Comment by Lxtoofonea** [comment]
+1, oh you and your nano suit

**Pull request merged**